src/content/en/2020/http2.md

```diff
@@ -66,7 +66,7 @@ Last year's analysis of HTTP Archive data showed that HTTP/2 was used for over 5
 )
 }}
 
-When comparing Figure 22.3 with last year's results, there has been a **10% increase in HTTP/2 requests** and a corresponding 10% decrease in HTTP/1 requests. This is the first year that gQUIC can be seen in the dataset.
+When comparing Figure 22.3 with last year's results, there has been a **10% increase in HTTP/2 requests** and a corresponding 10% decrease in HTTP/1.x requests. This is the first year that gQUIC can be seen in the dataset.
```

Comment:
I see a few more places where `HTTP/1` is used. Should all of those also be converted to `HTTP/1.x` for consistency?

Reply:
Yes, we should take a look and decide whether we're talking about both 1.0 and 1.1, or 1.1 specifically.